src/common/dataHooks/useSearchLoader.js

```diff
@@ -51,6 +51,7 @@ function useSearchLoader({ searchParams }) {
       sitename: result?.entry?.site?.slug,
       siteTitle: result?.entry?.site?.title,
       visibility: result?.entry?.visibility,
+      siteVisibility: result?.entry?.site.visibility,
```

Comment:
I believe the optional chaining operator (?) got missed here. The site will always be present, but good idea to add that operator to prevent any exceptions.